src/app/pipes/latitude.pipe.ts

```diff
@@ -0,0 +1,23 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'latitude'
+})
+export class LatitudePipe implements PipeTransform {
+
+  transform(value: any, args?: any): any {
+    /*
```

Comment:
Is there a reason to keep this?

Reply:
It's actually just a reminder to explain how the conversion works from *DD* to *DMS* coordinates. Not necessary but I found it makes the actual conversion code clearer !